fairmq/sdk/commands/Commands.h

```diff
@@ -314,11 +319,11 @@ struct Cmds
 
     Cmd& At(size_t i) { return *(fCmds.at(i)); }
 
-    size_t Size() { return fCmds.size(); }
+    size_t Size() const { return fCmds.size(); }
     void Reset() { fCmds.clear(); }
 
-    std::string Serialize() const;
-    void Deserialize(const std::string&);
+    std::string Serialize(const FormatType type = FormatType::Binary) const;
```

Comment:
```suggestion
    std::string Serialize(Format format = Format::Binary) const;
```

Just a minor comment to the naming: `fair::mq::sdk::cmd::FormatType::Binary` vs `fair::mq::sdk::cmd::Format::Binary`. Format*Type* feels *doppelgemoppelt* to me here.